bitswap/client/internal/messagequeue/donthavetimeoutmgr_test.go

```diff
@@ -72,6 +72,26 @@ func (tr *timeoutRecorder) clear() {
 	tr.timedOutKs = nil
 }
 
+// quartzAdd advances the time by the given duration through any events that
+// happen before or at the final time.
+//
+// TODO: Replace calls to quartzAdd with calls to clock.Advance and
+// clock.AdvanceNext to provide better control over time events.
+func quartzAdd(c *quartz.Mock, d time.Duration) {
+	for d > 0 {
+		next, ok := c.Peek()
+		if !ok || next >= d {
+			c.Advance(d)
+			time.Sleep(time.Millisecond)
+			return
+		}
+		<-c.Advance(next).Done()
```

Comment:
any reason not to use `MustWait`?

```suggestion
		c.Advance(next).MustWait(context.Background())
```

Reply:
Same as above.